config/interceptors/core-interceptors.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Interceptor
+metadata:
+  name: cel
```

Comment:
Putting them in namespace `tekton-pipelines` would make sense to me. This is where basically everything else is, right? Especially since they are shipped together.

Reply:
Right now this is a cluster wide type i.e. you install the InterceptorType GitHub for your entire cluster. Similar to how you install pipelines for the entire cluster.